src/UIAPIServer/handlers.ts

```diff
@@ -228,6 +239,9 @@ export const createHandlers = () => ({
     post: createDFSPCA,
     put: setDFSPCA,
   },
+  '/dfsp/endpoints': {
```

Comment:
Why do we need this operation?

Reply:
When I started testing all my changes locally, I saw lots of errors on _pm4ml_ ui. And I started fixing them.  But then I realized that there were to many similar errors, and i asked about it: https://infitx.slack.com/archives/C04HG5L8LDQ/p1696503562645689
So this fix was the first I made, but for now, seems, we can avoid it